bip-0077.mediawiki

```diff
@@ -0,0 +1,553 @@
+with multiple inputs "necessarily reveal that their inputs were owned by the same owner," by enabling two
+owners to provide input in a transaction.
+
+The Payjoin protocols automate cooperative transaction construction to break that common-input assumption.
+The increased opportunity to batch payments and execute transaction cut-through increases intent throughput,
+since multiple intents combined take up fewer bytes than independent transactions.
+
+Payjoin version 1's requirements have proven to be an obstacle to adoption. Version 1 coordinates payjoins
+over a public server endpoint secured by either TLS or Tor hidden service hosted by the receiver. Version 1
+is also synchronous, requiring both sender and receiver to be online simultaneously to payjoin. Both
+requirements present significant barriers for all but sophisticated server operators or those wallets with
+complex Tor integration. Wallet developers [[https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-
+January/018358.html| regard]] these as limits to Payjoin adoption.
+
+The primary goal of this proposal is to provide a practical coordination mechanism that can be implemented
+in a majority of bitcoin software environments. This is done here using a simple protocol built on bitcoin
+URI requests, web standards, common cryptography, and minimal dependencies.
+
+===Relation to BIP 78 (Payjoin version 1)===
+
+The message payloads in this version parallel those used in BIP 78, while being encapsulated in authenticated
+encryption. TLS and Tor security, which depend on third-party authorities, are replaced with Hybrid Public
+Key Encryption ([[https://www.rfc-editor.org/rfc/rfc9180| HPKE]]). The synchronous HTTP client-server model
+is replaced with an asynchronous store-and-forward mechanism.
+
+The BIP 78 standard allows for an [[https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#unsecured-
+payjoin-server| unsecured Payjoin server|]] to operate separately from the so-called "payment server"
+responsible for generating [[https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki| BIP 21]] request
+URIs. Because BIP 78 messages relayed over an unsecured server are neither end-to-end authenticated nor
+encrypted between sender and receiver, a malicious unsecured Payjoin server is able to modify the Proposal
+PSBT in flight, thus requiring payment output substitution to be disabled. Output substitution is useful for
+a number of block space optimizations, including payment batching and transaction cut-through. This proposal
+introduces authentication and encryption to secure output substitution by using a directory server without
+compromising sender or receiver privacy.
+
+Although unsecured Payjoin server separation is mentioned in BIP 78, no known specification or implementation
+exists. This document specifies a way to use the Payjoin directory as an unsecured backwards compatible server
```

Comment:
There are alternately references to "Payjoin directory", "Payjoin Directory", and "payjoin directory" right now. Does a Payjoin Directory technically qualify as a proper noun? Same question for OHTTP relay vs. OHTTP Relay.

Whichever the choice, this should be consistent